Modules/MUON/Common/src/TracksTask.cxx

```diff
@@ -144,32 +158,34 @@ void TracksTask::monitorData(o2::framework::ProcessingContext& ctx)
 {
   ILOG(Info, Devel) << "Debug: MonitorData" << ENDM;
 
-  mRecoCont.collectData(ctx, *mDataRequest.get());
-
-  ILOG(Info, Devel) << "Debug: Collected data" << ENDM;
-
   if (!assertInputs(ctx)) {
     return;
   }
 
+  mRecoCont.collectData(ctx, *mDataRequest.get());
+
+  ILOG(Info, Devel) << "Debug: Collected data" << ENDM;
+
   ILOG(Info, Devel) << "Debug: Asserted inputs" << ENDM;
```

Comment:
This message should go before the call to `mRecoCont.collectData()`

Reply:
done